src/index.html

```diff
@@ -17,28 +17,28 @@ <h2 id="app-message">A message for the user</h2>
         <h2>Daily Fitness</h2>
         <section class="card-container" id="daily-cards">
           <section class="card" id="hydration-today">
-            <p class="card-today-hydration">
+            <p class="hydration-card">
               You drank
               <span class="number" id="numOunces">0</span>
               oz today. <br />
-              Your average water intake is 
+              Your average water intake is
               <span class="number" id="numOunces-average"></span>
             </p>
             <button id="new-hydration">add new info</button>
           </section>
           <section class="card" id="activity-today">
-            <p class="card-today-activity">
+            <p class="activity-card">
               Your step count today:
               <span class="number" id="numSteps">0</span>
-              Your stair count today: 
+              Your stair count today:
               <span class="number" id="flightsOfStairs">0</span>
               Your minutes active today:
               <span class="number" id="minutesActive">0</span>
             </p>
             <button id="new-activity">add new info</button>
           </section>
           <section class="card" id="sleep-today">
-            <p class="card-today-sleep">
+            <p class="sleep-card">
```

Comment:
These are good naming conventions, great work. I actually know what it is doing now. Good Job!

Reply:
I agree! This is way more manageable. I do think eventually we'll want the card section to have a secondary class (maybe just sleep) and to cut the <p> tags from card elements. This is a great first step!